### ISSUE TITLE
Datetime subtraction

### ISSUE THREAD

**Issue by jurjendijkstra** [Riverside-Software/sonar-openedge#484]
sometimes I see in sourcecode: 

      duration = dtStop - dtStart.

The programmer obviously believed that the interval between two datetime variables can be calculated by simple subtraction. There is an outcome, but it is not always the correct duration, sometimes it is a nonsense value.

Programmer should use  duration = interval(dtStop, dtStart, "milliseconds").

**Comment by movedoa** [comment]
Just out of curiosity, do you have an example for a wrong calculation?

**Comment by jurjendijkstra** [comment]
hmmm, weird, I just tried to make a little .p to demonstrate it, but can't  :-( 
Typically, just like going to the doctor. When you are there, it suddenly does not hurt.
Perhaps in a different OpenEdge version, right now I am looking at 11.6.

But I am sure that we have had duration logs with weird numbers in them, and after I have searched for subtractions and replaced them by the interval function calls all durations are good.
